v2_jobs.go

```diff
@@ -79,13 +82,16 @@ func (s *JobStatus) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
-type Task struct {
-	ID       string `json:"uuid"`
-	ExitCode uint8  `json:"exit_code"`
-}
-
-func (s *JobsServiceOp) List(ctx context.Context, ID string, r *JobsListRequest) ([]Job, *Response, error) {
-	path := fmt.Sprintf("%s/%s", jobsBasePath, ID)
+func (s *JobsServiceOp) List(ctx context.Context,
+	ID string,
+	r *JobsListRequest,
+	detail bool) ([]Job, *Response, error) {
+	var path string
+	if detail {
+		path = fmt.Sprintf("%s/%s%s", jobsBasePath, ID, jobsDetailsQuery)
+	} else {
+		path = fmt.Sprintf("%s/%s", jobsBasePath, ID)
+	}
```

Comment:
If the request struct is modified as I suggested above:

```go
type JobsListRequest struct {
	Microservice string `json:"microservice,omitempty"`
	LinkID       string `json:"link_uuid,omitempty"`
	Name         string `json:"name,omitempty"`

	// Detailed request, encoded as a query parameter.
	Detailed bool `json:"-"`
}
```

Then you should be able to do this:

```go
func (s *JobsServiceOp) List(ctx context.Context, ID string, r *JobsListRequest) ([]Job, *Response, error) {
	path := fmt.Sprintf("%s/%s", jobsBasePath, ID)

	if r.Detailed {
		var err error
		path, err = addOption(path, "detailed", "")
		if err != nil {
			return nil, nil, err
		}
	}
	
	// ... the rest remains unmodified
}
```

And here's how `addOption` should look like:

```go
func addOption(s, key, value string) (string, error) {
	origURL, err := url.Parse(s)
	if err != nil {
		return "", err
	}

	queryValues := origURL.Query()
	queryValues.Set(key, value)

	origURL.RawQuery = queryValues.Encode()

	return origURL.String(), nil
}
```